Utils/ScriptUtil.py

```diff
@@ -47,31 +47,39 @@ def run_command(hutil, args, cwd, operation, extension_short_name, version, exit
                                  stderr=err_out)
         time.sleep(1)
         while child.poll() is None:
-            msg = LogUtil.get_formatted_log("Command is running...",
+            msg_with_cmd_output = LogUtil.get_formatted_log("Command is running...",
                                             LogUtil.tail(std_out_file), LogUtil.tail(err_out_file))
-            hutil.log(msg)
-            hutil.do_status_report(operation, 'transitioning', '0', msg)
+            msg_without_cmd_output = "Command is running... Stdout/Stderr omitted from output."
+            hutil.log_to_file(msg_with_cmd_output)
+            hutil.log_to_console(msg_without_cmd_output)
+
+            hutil.do_status_report(operation, 'transitioning', '0', msg_without_cmd_output)
             time.sleep(interval)
 
         exit_code = child.returncode
         if child.returncode and child.returncode != 0:
-            msg = LogUtil.get_formatted_log("Command returned an error.",
+            msg_with_cmd_output = LogUtil.get_formatted_log("Command returned an error.",
                                             LogUtil.tail(std_out_file), LogUtil.tail(err_out_file))
-            hutil.error(msg)
+            msg_without_cmd_output = "Command returned an error. Stdout/Stderr omitted from output."
```

Comment:
You can add the exit code to the `msg_without_cmd_output ` variable. Better for debuggability.

Reply:
Good point.